streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/util/EnvUtil.java

```diff
@@ -30,7 +30,7 @@ public static String getHostName() {
 
   public static void loadNativeLibraries() {
     JniUtils.loadLibrary(BinaryFileUtil.CORE_WORKER_JAVA_LIBRARY, true);
-    JniUtils.loadLibrary("streaming_java");
+    io.ray.streaming.runtime.util.JniUtils.loadLibrary("streaming_java");
```

Comment:
we can unify the `loadLibrary`.

Reply:
> we can unify the `loadLibrary`.

I'm afraid that's hard to unify loadlibrary in mobius repo.
We might modify java source code from ray repo since BinaryUtils uses own classloader to extract library so file, but it's not found to libstreaming_java.so because this so files move from ray_dist.jar to our isolated jar(raystreaming_dist.jar).

Reply:
@BalaBalaYi https://github.com/ray-project/mobius/issues/44 this issue might helpful to know these details.

Reply:
emm, i mean maybe we can unify the `JniUtils `. Extend some implements or reuse ray's `JniUtils `  in streaming's `JniUtils `. So the code here will be more clearly.

Reply:
> emm, i mean maybe we can unify the `JniUtils `. Extend some implements or reuse ray's `JniUtils ` in streaming's `JniUtils `. So the code here will be more clearly: we can know the issue above according to the method directly.

But these all are almost static functions, it can not be managed by inherited or interface somethings.